openslides/core/projector.py

```diff
@@ -139,3 +139,10 @@ class Message(ProjectorElement):
     def check_data(self):
         if self.config_entry.get('message') is None:
             raise ProjectorException('No message given.')
+
+
+class ListOfSpeakersOverlay(ProjectorElement):
+    """
+    List of speakers (last, current, next) on the projector. Rendered as overlay.
+    """
+    name = 'core/speakeroverlay'
```

Comment:
The problem is not just with my list of speakers overlay, but also with other, existing slides (which, as of now, are trying to load all users).

An example is elections. As an user without "can_see_user_names" you cannot see the candidates of an election, nor the results.